src/Sylius/Behat/Context/Api/Shop/RegistrationContext.php

```diff
@@ -251,6 +251,14 @@ public function iShouldBeSubscribedToTheNewsletter(): void
         Assert::true($this->responseChecker->getResponseContent($response)['subscribedToNewsletter']);
     }
 
+    /**
+     * @Then I should be on my account dashboard
+     * @Then I should be on registration thank you page
+     */
+    public function intentionallyLeftBlank(): void
```

Comment:
I would stick to the convention of naming methods like we do in other steps, so make 2 blanks or one universal like
```suggestion
    public function iShouldBeOn(): void
```
but still it's just a suggestion ✌🏽